### DISCUSSION TITLE
【求助】有没有人知道怎么办啊？

### DISCUSSION THREAD

**Discussion by Phaeton9528** [share121/inter-knot#348] [其他]
![image](https://github.com/user-attachments/assets/7ac9b181-766b-4b96-8c5f-e785a17f98e5)
我发现如果有人回复我回过的帖子，我的邮箱也会一直弹消息，有点烦，有没有人知道怎么让它停下啊？

**Comment by SAKURA-DD** [comment]
同，我也想问问QAQ

**Comment by paul-way52** [comment]
拒收试试？

**Comment by dongzhiaidao** [comment]
好像可以在贴子的旁边那个“通知”那里点拒收

**Comment by wugan-bot156** [comment]
试试这个
点击头像进github→再点击右边的大头像→点击左边的Notifications→找Automatically watch teams然后关掉

**Reply by wugan-bot156** [reply]
说错了是左边的大头像

**Comment by Phaeton9528** [comment]
弄好了，谢谢大佬

**Comment by Buer-Nahida** [comment]
见https://github.com/share121/inter-knot/discussions/512
